case-lib/hijack.sh

```diff
@@ -102,7 +102,9 @@ function func_exit_handler()
 
     stop_test || exit_status=1
 
-    storage_checks || exit_status=1
+    if [ ! "$SKIP_STORAGE_CHECKS" -eq 1 ]; then
```

Comment:
Nit
```suggestion
    [ "$SKIP_STORAGE_CHECKS" -eq 1 ] || storage_checks ||
        exit_status=1
```
The fewer negations the better and `!` can't be copied to the command-line.

Reply:
Change of mind sorry
```shell
if [ "$ENABLE_STORAGE_CHECKS" = '1' ]; then
   storage_checks || exit_status=1
fi
```
Zero negation; no need.